### ISSUE TITLE
Add a tokeniser for Japanese

### ISSUE THREAD

**Issue by ftyers** [omnilingo/omnilingo#69]

**Comment by ftyers** [comment]
I had a look at various Python packages, and it seems like this one is the easiest to install:
* https://github.com/taishi-i/nagisa

I've implemented it based on that.

```
>>> tokenisers.tokenise("自然消滅することは目に見えてるじゃん。", lang="jpn")
['自然', '消滅', 'する', 'こと', 'は', '目', 'に', '見え', 'てる', 'じゃん', '。']
```

**Comment by ftyers** [comment]
Fixed in d189c3f.
